bluesky/simple_scans.py

```diff
@@ -66,8 +67,8 @@ class _BundledScan:
 
     def __init__(self):
         # subs and sub_factories can be set individually per instance
-        self.subs = self.default_subs
-        self.sub_factories = self.default_sub_factories
+        self.subs = list(self.default_subs)
```

Comment:
DWIM is always safe in the args to the Run Engine or the simple scans.

I appreciate the downside of doing DWIM in the attributes. We could do the normalization in the setter. That means pushing it into ScanBase, which I like to keep as light as possible, but it might be the least evil thing to do.